mix.exs

```diff
@@ -21,6 +21,7 @@ defmodule CloudPubSub.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
+      {:mix_test_watch, "~> 1.0"},
```

Comment:
This should probably be set to `only: [:dev, :test]` and `targets: :host`

Reply:
I agree. I am not gonna make that change in this PR yet because I when those changes happen I want the person who does them to also check that consumers of this library who also specify mix_test_watch do not have to specify `override: true` because of conflicts with this library.

@amclain or @DanLindeman can you make an issue for this?

Reply:
https://github.com/verypossible-labs/cloud_pub_sub/issues/16

Reply:
https://github.com/verypossible-labs/cloud_pub_sub/issues/17